app/ui/browser/views/page/status.jsx

```diff
@@ -51,6 +55,13 @@ Status.displayName = 'Status';
 
 Status.propTypes = {
   page: PropTypes.object.isRequired,
+  statusText: PropTypes.string.isRequired,
 };
 
-export default Status;
+function mapStateToProps(state) {
+  return {
+    statusText: getStatusText(state),
```

Comment:
I think I just misunderstood this, we should only need one status text at a time, if `page` is already handled without mapping, my understanding is complete and this makes sense to me now